test/relations.jl

```diff
@@ -476,12 +718,16 @@ end
         PhasePartition(q_tot),
     ) ≈ T
 
-    # Exner function. FIXME: add correctness tests
+    # Test Exner function calculations
     p = FT(1.e5)
     q_tot = FT(0.23)
-    @test TD.exner_given_pressure(param_set, p, PhasePartition(q_tot)) isa
-          typeof(p)
+    q_pt = PhasePartition(q_tot)
+    @test TD.exner_given_pressure(param_set, p, q_pt) isa typeof(p)
+    @test TD.exner_given_pressure(param_set, p / 10, q_pt) ≈
+          TD.exner_given_pressure(param_set, p, q_pt) /
+          10^(gas_constant_air(param_set, q_pt) / cp_m(param_set, q_pt))
 
+    # Test humidity and mixing ratio calculations
     q_tot = 0.1
     q_liq = 0.05
     q_ice = 0.01
```

Comment:
Use better values? (I don't care much about these though, as it is just testing mixing ratios.)